native/acter/src/api/verification.rs

```diff
@@ -517,20 +525,20 @@ async fn request_verification_handler(
         request
             .accept_with_methods(methods)
             .await
-            .expect("Can't accept verification request");
+            .expect("Unable to accept verification request with methods");
```

Comment:
there is any type of reasons why this might fail - think network issues - so this should _not_ use `expect` but probably instead return an `Err` in those cases.

Reply:
`request_verification_handler()` is callback fn that is called using `tokio::spawn()`.
That is, `tokio::spawn()` is called in `RUNTIME.spawn()` and we can't wait for `tokio::spawn()` to be finished.
It means we needn't to receive return value from `request_verification_handler()`.
Then `request_verification_handler()` needn't to return `Result<>` and we needn't return `Err` for error case of `VerificationRequest::accept_with_methods()`.

Reply:
Appended error handling to verification callback.
https://github.com/acterglobal/a3/pull/1252/commits/0c03a76369d3c19272ec69587bd2b0663fe3e6f8
But I wonder how `tokio::spawn()` will handle the return value of verification callback.